README.md

```diff
@@ -22,9 +22,62 @@ Once you join the The [Presto Foundation Slack Workspace](https://join.slack.com
+| Shixuan Fan               |[shixuan-fan](https://github.com/shixuan-fan)        | Facebook    | 1 Jan, 2020  | TBD          |
+| Jiexi Lin                 |[jessesleeping](https://github.com/jessesleeping)    | Facebook    | 1 Jan, 2020  | TBD          |
+| Venki Korukanti           |[vkorukanti](https://github.com/vkorukanti)          | Uber        | 14 Feb, 2020  | TBD         |
+
+Per the [Technical Charter](CHARTER.md), when there are less than six related companies represented on the TSC, the employees of each company may only submit one vote on TSC decisions where voting is required.
+
+### Term of TSC voting members
+
+The Presto TSC seeks to balance the energy and enthusiasm of new members with the stability provided by long-term members.  As such, TSC voting members will serve three-year terms, after which they will be eligible for re-election.  There is no limit to the number of times an individual may run for an open seat.
+
+To provide balance, the terms of TSC members will be adjusted so that roughly 1/3 of seats are elected each year.  If the classes become unbalanced, then the TSC shall designate the appropriate number of seats as one-year terms at the first opportunity to fix the imbalance.
+
+For simplicity, membership terms will be aligned to the calendar year, and will run from January 1st to December 31st of the following year. Initially, the TSC shall randomly designate 1/3 of the seats to be one-year terms, 1/3 of the seats to be two-year terms, and the remainder shall be three-year terms.
+
+### Becoming a TSC Member
+
+The Technical Charter specifies that TSC voting members are initially the Presto committers.  Prior to each election, the TSC will determine the number of available seats and solicit nominations from the community.  The TSC has primary responsibility for ensuring there are sufficient candidates to fill each available seat.
+
+#### Eligibility
+
+Beginning with the 2021 election, any member of the technical community may stand for election to the TSC.
+
+#### Nominating procedure
+
+The TSC shall announce the number of the seats and the nominating procedures on the [presto-dev mailing list](https://lists.prestodb.io/g/presto-dev) by October 15th.
+
+Nominations and self-nominations shall be sent to the [presto-dev mailing list](https://lists.prestodb.io/g/presto-dev) by November 15th.
+
+#### Election procedure
+
+The existing Presto TSC members (including the class whose term expires at the end of the year) shall vote using the [Condorcet method](https://civs.cs.cornell.edu/) by December 1st.
+
+The incoming class shall be announced by December 15th, and their term will begin January 1st of the following year.
+
+### Changing the composition of the TSC
+
+The TSC may increase or reduce the number of voting members at election time, provided that an attempt is made to ensure the classes remain balanced.
```

Comment:
Just to confirm, this shouldn't restrict changing the TSC size at any other time, right? Per the charter, the TSC should have that right at any time.

Reply:
Yes, that's correct.  In §2(i), the TSC may set a new maximum limit of TSC members by supermajority vote.  The reason for including this was so that it would be clear this is a good time to have the discussion, although it seems like some additional detail and a link would be warranted here.  I'll push a minor change in a minute.

Reply:
Ok, new text is pushed and ready for review.